Microsoft/WebTemplateStudio
# Create Base Templates for Flask

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/27725

This PR adds base templates for a new Framework 'Flask' to the application (in preview mode).

**How to Check**
- Using the wizard, try to generate a Flask Application with any of the available front-end frameworks.
- The wizard should successfully generate the project for you.
- Go to the project and run: `yarn install` to install the dependencies
- Run `yarn start` to start the project
- Currently, only the blank page would work.
- Support for other pages would be added using composition templates in the coming PRs.

closes #704
